src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksNewVendorRequestsServiceImpl.java

```diff
@@ -99,23 +81,28 @@ private void processEachPaymentWorksNewVendorRequestIntoKFS(List<String> identif
             LOG.info("processEachPaymentWorksNewVendorRequestIntoKFS: Processing started for PMW-vendor-id=" + pmwNewVendorRequestId);
             PaymentWorksVendor stgNewVendorRequestDetailToProcess = getPaymentWorksWebServiceCallsService().obtainPmwNewVendorRequestDetailForPmwIdentifier(pmwNewVendorRequestId, reportData);
             if (canPaymentWorksNewVendorRequestProcessingContinueForVendor(stgNewVendorRequestDetailToProcess, reportData)) {
-                PaymentWorksVendor savedStgNewVendorRequestDetailToProcess = pmwPendingNewVendorRequestInitialSaveToKfsStagingTable(stgNewVendorRequestDetailToProcess);
+                PaymentWorksVendor savedStgNewVendorRequestDetailToProcess = null;
+                try {
+                    savedStgNewVendorRequestDetailToProcess = pmwPendingNewVendorRequestInitialSaveToKfsStagingTable( stgNewVendorRequestDetailToProcess);
+                } catch (Exception e) {
+                    LOG.error("processEachPaymentWorksNewVendorRequestIntoKFS, unable to save payment works vendor for pmwNewVendorRequestId: " + pmwNewVendorRequestId, e);
+                    reportData.getRecordsGeneratingExceptionSummary().incrementRecordCount();
+                    reportData.addPmwVendorThatCouldNotBeProcessed(new PaymentWorksBatchReportRawDataItem(pmwNewVendorRequestId, 
+                            getConfigurationService().getPropertyValueAsString(PaymentWorksKeyConstants.INITIAL_PAYMENT_WORKS_VENOR_SAVE_ERROR_MESSAGE)));
```

Comment:
I don't think you need to do the reportData updates within the catch() block, since the relevant updates should already be getting handled by the call to pmwNewVendorSaveToStagingTableWasSuccessful() call below.

Reply:
OK, I will remove the report data additions